idaes/core/ui/fsvis/static/js/stream_table.js

```diff
@@ -87,13 +154,34 @@ export class StreamTable {
                         row_object[variable_col] = row_object[variable_col] + '<span class="streamtable-units">&ndash;</span>';
                     }
                 }
-                else {
+                else if (columns[col_index] === "Variable") {
                     row_object[columns[col_index]] = data[col_index];
                 }
+                else {
+                    var [value, type] = data[col_index];
+                    switch (type) {
+                        case this.FIXED:
```

Comment:
See comment for L60 above, again you can compress to one body:
```
this.existing_var_types.add(type)
```

Reply:
Note that if you defined an object whose keys were these four values, you could also do:
```
if type in this.VAR_TYPES:
    this.existing_var_types.add(type)
else:
   console.warn(..) etc.
```
This would also simplify the other case statement(s).